### DISCUSSION TITLE
Relative images in README.md files do not show up

### DISCUSSION THREAD

**Discussion by Narigo** [community/community#72093] [Mobile]
### Select Topic Area

Bug

### Body

Hi GitHub mobile team!

We have linked images with relative paths (same directory) in a `README.md` file like this: `![image-alt-text](./the-image.png)`. On regular browsers, this seems to be the way to go and does show the correct images. See this URL: https://github.com/compose-us/build-in-public/blob/main/updates/2023-10-23%20Defining%20our%20mission%20-%20Improve%20Web%20Form%20File%20Uploads/README.md

Multiple people with the GitHub mobile app told me that they are unable to see those images though and provided a screenshot of the issue:

![GitHub mobile app not showing images in Markdown](https://github.com/community/community/assets/1767865/18a6a08a-e156-443b-abae-cf6d406f8bb6)

They can see the images when opening it in their regular browser though, so I suspect this being an issue with the GitHub mobile app itself.

Since I'm not using the app myself, is this reproducible for you or do you need additional information from the users? (I can ask them to provide feedback here)

Thank you!

**Comment by mattruggio** [comment]
Hey @Narigo, I believe you are correct: there looks to be some differences in our Markdown rendering pipeline between our web app and GraphQL API/mobile apps. I have forwarded this bug report to our Client Apps Platform team but cannot guarantee any specific priority on it. Thank you for your submission!

**Reply by Narigo** [reply]
Thanks for putting it on your list! If they have a workaround, it would be great to share, so at least I can fix it for the people who cannot see it yet.

**Reply by jolexxa** [reply]
Would also like to reiterate on this -- relative images are so common in README's that I think this is affecting a lot of people.

**Reply by AshishJii** [reply]
The issue is still here in the app. People usually surf and read repos in mobile app. Incomplete markdown support breaks the flow, making the app completely useless.

**Comment by Boneheinwai6801** [comment]
write and restart laptop in it
